### ISSUE TITLE
Error while running cppcheck

### ISSUE THREAD

**Issue by oedze** [Facthunder/cppcheck#34]
I have updated to the lastest 1.12.1 version, thanks for updating!

I am running into an issue though. 
Whenever i run the command `cppcheck` i get the following error: `cppcheck: error while loading shared libraries: libpcre.so.3: cannot open shared object file: No such file or directory`

It looks like the library is missing or something, but it is being installed. I can't seem to figure it out. 

I Tried this on both our gitlab runner which is running Linux and also on my windows desktop using docker desktop, in both cases i get this error.

What could this be?

**Comment by carlosperate** [comment]
Having the same issue with the latest v2.12.1 release.

**Comment by donaldmurf** [comment]
Also having this issue on the same release (v2.12.1).

**Comment by campbell-brown** [comment]
I am having the same issue as well. To reproduce:
```console
$ docker run --rm -v .:/src ghcr.io/facthunder/cppcheck:2.12.1 cppcheck -v
cppcheck: error while loading shared libraries: libpcre.so.3: cannot open shared object file: No such file or directory
```

**Comment by begarco** [comment]
Fixed in new `2.12.1`.
